app/javascript/components/admin/UserModal.js

```diff
@@ -75,9 +77,11 @@ class UserModal extends React.Component {
                     ? { label: this.props.initialUserData.jurisdiction_path, value: this.props.initialUserData.jurisdiction_path }
                     : { label: this.props.jurisdiction_paths[0], value: this.props.jurisdiction_paths[0] }
                 }
-                options={this.props.jurisdiction_paths.map(path => {
-                  return { label: path, value: path };
-                })}
+                options={this.props.jurisdiction_paths
+                  .map(path => {
+                    return { label: path, value: path };
```

Comment:
So im totally okay with leaving this as it's formatted now, but, just so you know, you can write inline single-line object returns in the format of 
`.map(path => ({label: path, value: path}))`

I don't often use it cause it's not as easily readable or debugg-able, but it's good to know!